### PR TITLE
Nginx hotfix to remove HTTPS for debug mode

### PR DESCRIPTION
**Pull request by CoolNamesAllTaken**

### CODE REVIEW

File: code/svb-docker/python/svb_web/core/tests/test_debit_card.py
Comment (cahogan) [inline code/svb-docker/python/svb_web/core/tests/test_debit_card.py:1]: Doesn't belong in this PR

File: code/svb-docker/python/svb_web/core/utils/debit_card.py
Comment (cahogan) [inline code/svb-docker/python/svb_web/core/utils/debit_card.py:1]: Doesn't belong in this PR

File: code/svb-docker/nginx/debug/app.conf [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+#   return 301 https://$host$request_uri;
+# }
+
+## svb.pantsforbirds.com
+# HTTP access
+server {
+  listen 80; # ipv4
+  listen [::]:80; # ipv6
+  server_name svb.pantsforbirds.com www.svb.pantsforbirds.com;
```
Comment (cahogan) [inline code/svb-docker/nginx/debug/app.conf:19]: Is this correct?

**Review by cahogan — APPROVED** [review]
Had some questions and NPs but approved.

File: code/svb-docker/nginx/debug/app.conf (hunk 1 above)
Comment (CoolNamesAllTaken) [reply code/svb-docker/nginx/debug/app.conf:19]: Yes? Specific concerns?
